Validator/IsInt.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Validator
+{
+    public partial class Validator
+    {
+        /// <summary>
+        /// Determine whether input represents an integer
+        /// Can only check numbers as large as long.MaxValue or as small as long.MinValue
+        /// Commas are not valid
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public static bool IsInt(string input)
+        {
+            long dummy;
+            return long.TryParse(input, out dummy);
```

Comment:
@mgroves "Int" in `IsInt()` has special meaning in C#, I'd have expected this to be `int.TryParse()`. Are you trying to reconcile the difference with JS by calling it `IsInt()` but having it really mean--in a C# context--`IsLong()`?